python/adbc_driver_manager/tests/test_lowlevel.py

```diff
@@ -390,3 +390,57 @@ def test_child_tracking(sqlite):
                 RuntimeError, match="Cannot close AdbcDatabase with open AdbcConnection"
             ):
                 db.close()
+
+
+@pytest.mark.sqlite
+def test_pycapsule(sqlite):
```

Comment:
I further expanded the specific test that David started, but in addition (at least if requiring pyarrow>=14 for testing) I could also update the other tests above to do the import/export using capsules instead of the current calls to `_import_from_c`), that then will also automatically give some more test coverage.